art/attacks/inference/membership_inference/label_only_boundary_distance.py

```diff
@@ -196,6 +202,62 @@ def calibrate_distance_threshold(
 
         self.distance_threshold_tau = distance_threshold_tau
 
+    def calibrate_distance_threshold_unsupervised(self, top_t: int, num_samples: int, max_queries: int, **kwargs):
+        """
+        Calibrate distance threshold on randomly generated samples, choosing the top-t percentile of the noise needed
+        to change the classifier's initial prediction.
+
+        | Paper link: https://arxiv.org/abs/2007.15528
+
+        :param top_t: Top-t percentile.
+        :param num_samples: Number of random samples to generate.
+        :param max_queries: Maximum number of queries.
```

Comment:
Isn't this number actually multiplied by the num_queries param supplied to hsj? I think this should be clarified...

Reply:
Do you mean the `max_iter` parameter of hsj? You are right, the maximum number of iterations one does with hsj on a given sample is `max_queries * max_iter`. Do you want me to clarify this?

Reply:
Yes, this is what I meant. You can just add another sentence to the end of the parameter description.